### PR TITLE
[warrior] Weekly update

### PR DESCRIPTION
**Pull request by dothanhtrung**

### CODE REVIEW

File: recipes-debian/util-linux/util-linux_debian.bb [hunk 1]
```diff
@@ -27,3 +27,8 @@ SRC_URI += "file://configure-sbindir.patch \
             file://avoid_parallel_tests.patch \
             file://check-for-_HAVE_STRUCT_TERMIOS_C_OSPEED.patch \
 "
+
+do_install_append() {
+	# Correct owner after moving files
+	chown -R root:root ${D}${base_sbindir} ${D}${base_bindir}
```
Comment (yuwata) [inline recipes-debian/util-linux/util-linux_debian.bb]: I think this is not necessary if #187 is merged.

Comment (dothanhtrung) [reply recipes-debian/util-linux/util-linux_debian.bb]: Thank you. Do you mean commit of pseudo? I will try it.

Comment (yuwata) [reply recipes-debian/util-linux/util-linux_debian.bb]: Yes.

Comment (dothanhtrung) [reply recipes-debian/util-linux/util-linux_debian.bb]: I tried to rebuild package several time and this problem still exists after #187 is merged. It often happens when I change `MACHINE` in same build directory.

**Comment by dothanhtrung** [comment]
Dropped commit "meson: Handle strings in cross file args". This commit will be in another PR.

File: recipes-debian/util-linux/util-linux_debian.bb (hunk 1 above)
Comment (zuka0828) [reply recipes-debian/util-linux/util-linux_debian.bb]: IMU, the owner of installed files should be determined by the original do_install(). If the warning randomly happens, it's better to check the differences between the normal/abnormal cases.

@dothanhtrung  I think more time is required to fix this issue, so how about dropping it and merge other commits, then apply this or updated patch for util-linux later?

Comment (dothanhtrung) [reply recipes-debian/util-linux/util-linux_debian.bb]: Okay, that's better. I dropped this commit. I will try to find a way to reproduce this warning.

Comment (zuka0828) [reply recipes-debian/util-linux/util-linux_debian.bb]: @dothanhtrung Thank you

**Pull request merged**